process-lib/src/process.ml

```diff
@@ -657,6 +661,47 @@ let run_bool ?(true_v=[0]) ?(false_v=[1]) prog args =
     Printf.ksprintf failwith "Command exited with unexpected code %d: %s"
       code (cmd_line prog args)
 
+let run_exit_status =
+  let cmd = command_exit_status "run" in
+  fun prog args -> cmd prog args
+    
+let run_exit_code =
+  let cmd = command_exit_code "run" in
+  fun prog args -> cmd prog args
+
+let run =
+  let cmd = command "run" in
+  fun prog args -> cmd prog args
+
+let run_bool =
+  let cmd = command_bool "run" in
+  fun ?(true_v=[0]) ?(false_v=[1]) prog args -> cmd ~true_v ~false_v prog args
```

Comment:
Can you use `fun ?true_v ?false_v prog args -> cmd ?true_v ?false_v prog args` to avoid copying the default values?